app/models/log_event.rb

```diff
@@ -27,7 +27,11 @@ def loggable_to_s
     when AccountUser
       "#{loggable.account} / #{loggable.user}"
     else
-      loggable.to_s
+      if loggable.respond_to?(:to_log_s)
+        loggable.to_log_s
+      else
+        loggable.to_s
+      end
```

Comment:
Can we pull the `AccountUser` case into `AccoutnUser#to_log_s` and then get rid of the `case` completely?

Reply:
Sure I'm on it